### PR TITLE
Added ZdsToZgw files

### PR DESCRIPTION
**Pull request by smisidjan**

### CODE REVIEW

**Review by bbrands02 — APPROVED** [review]

File: Installation/Action/zds.ZdsDocumentAction.action.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
+{
+  "title": "ZdsDocumentAction",
+  "$id": "https://zds.nl/action/zds.ZdsDocumentAction.action.json",
+  "$schema": "https://docs.commongateway.nl/schemas/ActionHandler.schema.json",
```
Comment (WilcoLouwerse) [inline Installation/Action/zds.ZdsDocumentAction.action.json:4]: should be
https://docs.commongateway.nl/schemas/Action.schema.json

File: Installation/Action/zds.ZdsDocumentIdentificationAction.action.json [hunk 2]
```diff
@@ -0,0 +1,19 @@
+{
+  "title": "ZdsDocumentIdentificationAction",
+  "$id": "https://zds.nl/action/zds.ZdsDocumentIdentificationAction.action.json",
+  "$schema": "https://docs.commongateway.nl/schemas/ActionHandler.schema.json",
```
Comment (WilcoLouwerse) [inline Installation/Action/zds.ZdsDocumentIdentificationAction.action.json:4]: should be
https://docs.commongateway.nl/schemas/Action.schema.json

File: Installation/Action/zds.ZdsZaakAction.action.json [hunk 3]
```diff
@@ -0,0 +1,19 @@
+{
+  "title": "ZdsZaakAction",
+  "$id": "https://zds.nl/action/zds.ZdsZaakAction.action.json",
+  "$schema": "https://docs.commongateway.nl/schemas/ActionHandler.schema.json",
```
Comment (WilcoLouwerse) [inline Installation/Action/zds.ZdsZaakAction.action.json:4]: should be
https://docs.commongateway.nl/schemas/Action.schema.json

File: Installation/Action/zds.ZdsZaakIdentificationAction.action.json [hunk 4]
```diff
@@ -0,0 +1,19 @@
+{
+  "title": "ZdsZaakIdentificationAction",
+  "$id": "https://zds.nl/action/zds.ZdsZaakIdentificationAction.action.json",
+  "$schema": "https://docs.commongateway.nl/schemas/ActionHandler.schema.json",
```
Comment (WilcoLouwerse) [inline Installation/Action/zds.ZdsZaakIdentificationAction.action.json:4]: should be
https://docs.commongateway.nl/schemas/Action.schema.json

File: Installation/Mapping/zds.extraElementToEigenschap.mapping.json [hunk 5]
```diff
@@ -0,0 +1,11 @@
+{
+  "title": "ZDSToZGWZaak",
+  "$id": "https://zds.nl/mapping/zds.extraElementToEigenschap.mapping.json",
+  "$schema": "https://json-schema.org/draft/2020-12/mapping",
```
Comment (WilcoLouwerse) [inline Installation/Mapping/zds.extraElementToEigenschap.mapping.json:4]: Should be:
https://docs.commongateway.nl/schemas/Mapping.schema.json

File: Installation/Mapping/zds.extraElementToZaakEigenschap.mapping.json [hunk 6]
```diff
@@ -0,0 +1,15 @@
+{
+  "title": "ZDSToZGWZaak",
+  "$id": "https://zds.nl/mapping/zds.extraElementToZaakEigenschap.mapping.json",
+  "$schema": "https://json-schema.org/draft/2020-12/mapping",
```
Comment (WilcoLouwerse) [inline Installation/Mapping/zds.extraElementToZaakEigenschap.mapping.json:4]: Should be:
https://docs.commongateway.nl/schemas/Mapping.schema.json

File: Installation/Mapping/zds.isVanToZgwZaaktype.mapping.json [hunk 7]
```diff
@@ -0,0 +1,11 @@
+{
+  "title": "ZDSToZGWZaakType",
+  "$id": "https://zds.nl/mapping/zds.isVanToZgwZaaktype.mapping.json",
+  "$schema": "https://json-schema.org/draft/2020-12/mapping",
```
Comment (WilcoLouwerse) [inline Installation/Mapping/zds.isVanToZgwZaaktype.mapping.json:4]: Should be:
https://docs.commongateway.nl/schemas/Mapping.schema.json

File: Installation/Mapping/zds.zdsDocumentIdToZgwDocument.mapping.json [hunk 8]
```diff
@@ -0,0 +1,10 @@
+{
+  "title": "ZDSToZGWZaak",
+  "$id": "https://zds.nl/mapping/zds.zdsDocumentIdToZgwDocument.mapping.json",
+  "$schema": "https://json-schema.org/draft/2020-12/mapping",
```
Comment (WilcoLouwerse) [inline Installation/Mapping/zds.zdsDocumentIdToZgwDocument.mapping.json:4]: Should be:
https://docs.commongateway.nl/schemas/Mapping.schema.json

File: Installation/Mapping/zds.zdsDocumentToZgwDocument.mapping.json [hunk 9]
```diff
@@ -0,0 +1,20 @@
+{
+  "title": "ZDSToZGWDocument",
+  "$id": "https://zds.nl/mapping/zds.zdsDocumentToZgwDocument.mapping.json",
+  "$schema": "https://json-schema.org/draft/2020-12/mapping",
```
Comment (WilcoLouwerse) [inline Installation/Mapping/zds.zdsDocumentToZgwDocument.mapping.json:4]: Should be:
https://docs.commongateway.nl/schemas/Mapping.schema.json

File: Installation/Mapping/zds.zdsHeeftAlsInitiatorToRol.mapping.json [hunk 10]
```diff
@@ -0,0 +1,33 @@
+{
+  "title": "ZDSToZGWZaak",
+  "$id": "https://zds.nl/mapping/zds.zdsHeeftAlsInitiatorToRol.mapping.json",
+  "$schema": "https://json-schema.org/draft/2020-12/mapping",
```
Comment (WilcoLouwerse) [inline Installation/Mapping/zds.zdsHeeftAlsInitiatorToRol.mapping.json:4]: Should be:
https://docs.commongateway.nl/schemas/Mapping.schema.json

File: Installation/Mapping/zds.zdsHeeftAlsInitiatorToRolType.mapping.json [hunk 11]
```diff
@@ -0,0 +1,11 @@
+{
+  "title": "ZDSToZGWZaak",
+  "$id": "https://zds.nl/mapping/zds.zdsHeeftAlsInitiatorToRolType.mapping.json",
+  "$schema": "https://json-schema.org/draft/2020-12/mapping",
```
Comment (WilcoLouwerse) [inline Installation/Mapping/zds.zdsHeeftAlsInitiatorToRolType.mapping.json:4]: Should be:
https://docs.commongateway.nl/schemas/Mapping.schema.json

File: Installation/Mapping/zds.zdsZaakIdToZgwZaak.mapping.json [hunk 12]
```diff
@@ -0,0 +1,10 @@
+{
+  "title": "ZDSToZGWZaak",
+  "$id": "https://zds.nl/mapping/zds.zdsZaakIdToZgwZaak.mapping.json",
+  "$schema": "https://json-schema.org/draft/2020-12/mapping",
```
Comment (WilcoLouwerse) [inline Installation/Mapping/zds.zdsZaakIdToZgwZaak.mapping.json:4]: Should be:
https://docs.commongateway.nl/schemas/Mapping.schema.json

File: Installation/Mapping/zds.zdsZaakToZgwZaak.mapping.json [hunk 13]
```diff
@@ -0,0 +1,29 @@
+{
+  "title": "ZDSToZGWZaak",
+  "$id": "https://zds.nl/mapping/zds.zdsZaakToZgwZaak.mapping.json",
+  "$schema": "https://json-schema.org/draft/2020-12/mapping",
```
Comment (WilcoLouwerse) [inline Installation/Mapping/zds.zdsZaakToZgwZaak.mapping.json:4]: Should be:
https://docs.commongateway.nl/schemas/Mapping.schema.json

File: Installation/Mapping/zds.zgwDocumentToBv03.mapping.json [hunk 14]
```diff
@@ -0,0 +1,17 @@
+{
+  "title": "ZGWZaakToDu02",
+  "$id": "https://zds.nl/mapping/zds.zgwDocumentToBv03.mapping.json",
+  "$schema": "https://json-schema.org/draft/2020-12/mapping",
```
Comment (WilcoLouwerse) [inline Installation/Mapping/zds.zgwDocumentToBv03.mapping.json:4]: Should be:
https://docs.commongateway.nl/schemas/Mapping.schema.json

File: Installation/Mapping/zds.zgwDocumentToDu02.mapping.json [hunk 15]
```diff
@@ -0,0 +1,21 @@
+{
+  "title": "ZGWZaakToDu02",
+  "$id": "https://zds.nl/mapping/zds.zgwDocumentToDu02.mapping.json",
+  "$schema": "https://json-schema.org/draft/2020-12/mapping",
```
Comment (WilcoLouwerse) [inline Installation/Mapping/zds.zgwDocumentToDu02.mapping.json:4]: Should be:
https://docs.commongateway.nl/schemas/Mapping.schema.json

File: Installation/Mapping/zds.zgwZaakToBv03.mapping.json [hunk 16]
```diff
@@ -0,0 +1,17 @@
+{
+  "title": "ZGWZaakToDu02",
+  "$id": "https://zds.nl/mapping/zds.zgwZaakToBv03.mapping.json",
+  "$schema": "https://json-schema.org/draft/2020-12/mapping",
```
Comment (WilcoLouwerse) [inline Installation/Mapping/zds.zgwZaakToBv03.mapping.json:4]: Should be:
https://docs.commongateway.nl/schemas/Mapping.schema.json

File: Installation/Mapping/zds.zgwZaakToDu02.mapping.json [hunk 17]
```diff
@@ -0,0 +1,21 @@
+{
+  "title": "ZGWZaakToDu02",
+  "$id": "https://zds.nl/mapping/zds.zgwZaakToDu02.mapping.json",
+  "$schema": "https://json-schema.org/draft/2020-12/mapping",
```
Comment (WilcoLouwerse) [inline Installation/Mapping/zds.zgwZaakToDu02.mapping.json:4]: Should be:
https://docs.commongateway.nl/schemas/Mapping.schema.json

File: src/ActionHandler/DocumentIdentificatieActionHandler.php [hunk 18]
```diff
@@ -0,0 +1,62 @@
...
+    /**
+     *  This function returns the required configuration as a [json-schema](https://json-schema.org/) array.
+     *
+     * @return array a [json-schema](https://json-schema.org/) that this  action should comply to
+     */
+    public function getConfiguration(): array
+    {
+        return [
+            '$id'         => 'https://zds.nl/zds.creerdocumentid.handler.json',
+            '$schema'     => 'https://json-schema.org/draft/2020-12/schema',
```
Comment (WilcoLouwerse) [inline src/ActionHandler/DocumentIdentificatieActionHandler.php:39]: Should be:
https://docs.commongateway.nl/schemas/ActionHandler.schema.json

File: src/ActionHandler/ZaakIdentificatieActionHandler.php [hunk 19]
```diff
@@ -0,0 +1,62 @@
...
+    /**
+     *  This function returns the required configuration as a [json-schema](https://json-schema.org/) array.
+     *
+     * @throws array a [json-schema](https://json-schema.org/) that this  action should comply to
+     */
+    public function getConfiguration(): array
+    {
+        return [
+            '$id'         => 'https://zds.nl/zds.creerzaakid.handler.json',
+            '$schema'     => 'https://json-schema.org/draft/2020-12/schema',
```
Comment (WilcoLouwerse) [inline src/ActionHandler/ZaakIdentificatieActionHandler.php:39]: Should be:
https://docs.commongateway.nl/schemas/ActionHandler.schema.json

File: src/ActionHandler/ZdsDocumentActionHandler.php [hunk 20]
```diff
@@ -0,0 +1,62 @@
...
+    /**
+     *  This function returns the required configuration as a [json-schema](https://json-schema.org/) array.
+     *
+     * @return array a [json-schema](https://json-schema.org/) that this  action should comply to
+     */
+    public function getConfiguration(): array
+    {
+        return [
+            '$id'         => 'https://zds.nl/zds.creerdocument.handler.json',
+            '$schema'     => 'https://json-schema.org/draft/2020-12/schema',
```
Comment (WilcoLouwerse) [inline src/ActionHandler/ZdsDocumentActionHandler.php:39]: Should be:
https://docs.commongateway.nl/schemas/ActionHandler.schema.json

File: src/ActionHandler/ZdsZaakActionHandler.php [hunk 21]
```diff
@@ -0,0 +1,62 @@
...
+    /**
+     *  This function returns the required configuration as a [json-schema](https://json-schema.org/) array.
+     *
+     * @return array a [json-schema](https://json-schema.org/) that this  action should comply to
+     */
+    public function getConfiguration(): array
+    {
+        return [
+            '$id'         => 'https://zds.nl/zds.creerzaak.handler.json',
+            '$schema'     => 'https://json-schema.org/draft/2020-12/schema',
```
Comment (WilcoLouwerse) [inline src/ActionHandler/ZdsZaakActionHandler.php:39]: Should be:
https://docs.commongateway.nl/schemas/ActionHandler.schema.json

**Review by WilcoLouwerse — CHANGES_REQUESTED** [review]

**Review by WilcoLouwerse — COMMENTED** [review]
approved

**Pull request merged**